.ci/jobs/elastic-agent-system-metrics-mbp.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - job:
-    name: "elastic-agent-system-metrics-mbp"
+    name: "elastic-agent/elastic-agent-system-metrics-mbp"
```

Comment:
The Multibranch Pipeline job will be created in https://fleet-ci.elastic.co/job/elastic-agent/